include/clover_field.h

```diff
@@ -6,6 +6,14 @@
 
 namespace quda {
 
+  // Prefetch type
```

Comment:
For an internal enum I wouldn't bother with the `Quda` prefix

Reply:
For an internal enum I would suggest to always use enum classes. No need to stick to plain old C. https://stackoverflow.com/questions/18335861/why-is-enum-class-preferred-over-plain-enum